homeassistant/components/media_player/device_condition.py

```diff
@@ -23,7 +24,14 @@
 
 from .const import DOMAIN
 
-CONDITION_TYPES = {"is_on", "is_off", "is_idle", "is_paused", "is_playing"}
+CONDITION_TYPES = {
+    "is_on",
+    "is_off",
+    "is_buffering",
```

Comment:
Don't forget the translation string.

Reply:
Added, including those from https://github.com/home-assistant/core/pull/70864